api/src/constants/index.js

```diff
@@ -86,5 +86,8 @@ export const AUDIT_ACTIONS = {
     audits: {
       list: 'audits.list',
     },
+    user: {
+      getUserPendingInvitations: 'user.getUserPendingInvitations'
```

Comment:
since the top level object describes the resource i'd probably go
```
{
  user: {
     invitations: {
        get: 'user.invitations.get'
     }

}
```

Reply:
fixed.